code/modules/mob/living/carbon/human/species/station/traits_vr/neutral.dm

```diff
@@ -80,6 +80,26 @@
 	H.verbs |= /mob/living/carbon/human/proc/succubus_drain_finalize
 	H.verbs |= /mob/living/carbon/human/proc/succubus_drain_lethal
 
+/datum/trait/resizing_immunity
```

Comment:
Resize immunity doesn't really need to be a trait; this locks it to custom species only. just add it to everyone.

Reply:
It is much-much-much simpler to code it as a trait in the meanwhile. Coding it in vore panel preferences is, first, much more complex task for me, since I have to mess with tgui, and second, this option is unrelated to vore, while everything else is.

Another option is make it a client pref, but client prefs are for sounds, visuals and interface, not for how your character interacts with stuff mechanically.

Reply:
Could make it an IC/OOC verb just fine. We already have verbs that toggle IC interactions.

Reply:
Still, right now would rather leave it as trait, since easier to code and does not create any additional buttons where they shouldn't be. Later, if needed, it can be redone as button - there are not much to change. Can be even left as species trait AND as pref button later.